pipelines/build/prTester/README.md

```diff
@@ -0,0 +1,127 @@
+
+  - Example of a pr that is in the queue:
+  ![Image of queued tester](../../../docs/images/pr_tester_queued.png)
+  - One that is at the front of the queue and currently being tested:
+  ![Image of building tester](../../../docs/images/pr_tester_building.png)
+
+- When the tester is done, it will return a response comment to the pr with feedback on the testing similar to the following:
+![Image of test result](../../../docs/images/pr_tester_result.png)
+
+- The message will vary depending on the result of the test. Please remember however, that failed tests may be due to existing problems in the nightly builds. If you're unsure if the tests failed because of your changes or not, check our [issue board](https://github.com/AdoptOpenJDK/openjdk-build/issues) and our [triage doc](https://docs.google.com/document/d/1vcZgHJeR8rW8U8OD23Uob7A1dbLrtkURZUkinUp7f_w/edit?usp=sharing) for the existing error
+
+  - 🟢**SUCCESS** 🟢 All the tests and jobs passed, congratulations!
+  - 🟠**FAILURE** 🟠 Some of the tests or jobs failed OR the job was aborted. Check the link in the field at the bottom of the PR for the job link to see exactly where it went wrong.
+  - 🔴**ERROR** 🔴 Something more serious went wrong with the tester itself. Please raise an issue with a link to the job, the error encountered and your PR that caused it (again, you can use the link at the bottom to see exactly what happened).
+
+- NOTE: [openjdk-build-compile-groovy](#openjdk-build-compile-groovy) does not produce these status messages for the reasons stated in [this issue](https://github.com/AdoptOpenJDK/openjdk-build/issues/2055#issuecomment-688802783).
+
+### `add to whitelist`
+
+- **ADMIN COMMAND ONLY**
+- This command adds a new user to the whitelist but not to the admin list. As of typing this, there is [currently no way to check if you have the correct permissions](https://github.com/AdoptOpenJDK/openjdk-build/issues/2055#issuecomment-688801090) to execute this command (or even run the tester yourself) so in absence of that, here is the current list of admins and whitelisted users:
+
+#### Admins
+
+  - [@gdams](https://github.com/gdams)
+  - [@johnoliver](https://github.com/johnoliver)
+  - [@jerboaa](https://github.com/jerboaa)
+  - [@karianna](https://github.com/karianna)
+  - [@sophia-guo](https://github.com/sophia-guo)
+  - [@M-Davies](https://github.com/M-Davies)
+
+#### Whitelist
+
+- Should you want to be promoted to the whitelist, please contact one of the admins through [#infrastructure](https://adoptopenjdk.slack.com/archives/C53GHCXL4) in Slack.
+
+  - [@jennli](https://github.com/jennli)
+  - [@douph1](https://github.com/douph1)
+  - [@sxa](https://github.com/sxa)
+  - [@aahlenst](https://github.com/aahlenst)
+  - [@austin0](https://github.com/austin0)
+  - [@andrew-m-leonard](https://github.com/andrew-m-leonard)
```

Comment:
We'd have to regularly update this list. I'd rather leave it out, especially because we have lookup instructions.